vmdb/Gemfile

```diff
@@ -11,9 +11,13 @@ gem "jquery-rjs", "=0.1.1", :git => 'https://github.com/amatsuda/jquery-rjs.git'
 gem 'angularjs-rails', '=1.2.4'
 gem 'jquery-rails', "=2.1.4"
 
+gem "win32-process", "~>0.8.0", :require => false, :platforms => [:mswin, :mingw]
```

Comment:
Looks like it does. I ran it on my Mac and it showed up. The issue appears to be contentious in general:

https://github.com/bundler/bundler-features/issues/4

I could move that into the ENV['APPLIANCE'] block as you suggested. Another option is to simply wrap that bit of code like so:

    if Gem.win_platform?
      gem "win32-process", "~>0.8.0", :require => false, :platforms => [:mswin, :mingw]
    end

Doing it that way prevents it from ending up in the Gemfile.lock.

If you think it should be moved into the ENV['APPLIANCE'] block, I'll need some guidance for how you want it done exactly.